main/forum/index.php

```diff
@@ -313,41 +307,68 @@ function hidecontent(content){
                         array(),
                         ICON_SIZE_SMALL
                     ).'</a>';
-                display_visible_invisible_icon(
-                    'forumcategory',
-                    strval(intval($forumCategory['cat_id'])),
-                    strval(intval($forumCategory['visibility']))
-                );
-                display_lock_unlock_icon(
-                    'forumcategory',
-                    strval(intval($forumCategory['cat_id'])),
-                    strval(intval($forumCategory['locked']))
-                );
-                display_up_down_icon(
-                    'forumcategory',
-                    strval(intval($forumCategory['cat_id'])),
-                    $forumCategories
+                    $iconsEdit .=  return_visible_invisible_icon(
+                        'forumcategory',
+                        strval(intval($idCategory)),
+                        strval(intval($forumCategory['visibility']))
+                    );
+                    $iconsEdit .= return_lock_unlock_icon(
+                        'forumcategory',
+                        strval(intval($$idCategory)),
+                        strval(intval($forumCategory['locked']))
+                    );
+                    $iconsEdit .= return_up_down_icon(
+                        'forumcategory',
+                        strval(intval($idCategory)),
+                        $forumCategories
+                    );
+                }
+            }
+
+            $html .= '<div class="category-forum">';
+            $html .= Display::tag(
+                    'div',
+                    $iconsEdit,
+                    array(
+                        'class' => 'pull-right'
+                    )
                 );
+            $icoCategory = Display::return_icon(
+                    'forum_blue.png',
+                    get_lang(
+                        $forumCategory['cat_title']
+                    ),
+                    array(),
+                    ICON_SIZE_MEDIUM
+            );
+            $html .= '<h3>' . $icoCategory . $titleCategory . '</h3>';
+
+            if ($descriptionCategory != '' && trim($descriptionCategory) != '&nbsp;')
+            {
+                $html .= '<div class="forum-description">'.$descriptionCategory.'</div>';
             }
-        }
-        echo '</th>';
-        echo '</tr>';
-        echo '</thead>';
 
+            $html .= '</div>';
+            echo $html;
+
+
+
+        echo '<div class="forum_display">';
```

Comment:
Hay algo raro en el alineamiento entre las líneas 352 y 356. Si es que no hay cambio de nivel de condición (if, else, while, for, etc), entonces estos dos "echo" deberían estar alineados...